src/codeu/chat/client/commandline/Chat.java

```diff
@@ -14,380 +14,383 @@
 
 package codeu.chat.client.commandline;
 
-import java.util.HashMap;
-import java.util.Map;
-import java.util.Scanner;
-import java.util.Stack;
+import java.io.IOException;
+import java.util.*;
```

Comment:
Wildcard imports are not considered good style: https://google.github.io/styleguide/javaguide.html#s3.3.1-wildcard-imports

import exactly the classes that you need.